geci_distance/geci_distance.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import pandas as pd
+import numpy as np
+from lmfit import Model
+from scipy import integrate
+
+
+def hazard_model(x, sigma, beta):
+    return 1 - np.exp(-((x / sigma) ** (-beta)))
+
+
+class GECI_Distance:
+    def __init__(self, n_obs, distances, n_bins=10):
+        self.distances = distances
+        self.n_bins = n_bins
+        self.n_obs = n_obs
```

Comment:
```suggestion
        self.area = None
        self.beta = None
        self.bins_mid_points = None
        self.detection_probability = None
        self.distances = distances
        self.length = None
        self.n_bins = n_bins
        self.n_obs = n_obs
        self.n_total = None
        self.norm_hist = None
        self.sigma = None
        self.width = None
```

Podríamos definir todos los atributos en el `__init__`?